### PR TITLE
security policy graduation to v1

### PR DESCRIPTION
**Pull request by aryan16**
This PR involves API changes required to graduate security policies from V1beta1 to V1 and some documentation changes for authorization policy.

### CODE REVIEW

File: security/v1/jwt.proto [hunk 1]
```diff
@@ -0,0 +1,157 @@
+// Copyright 2019 Istio Authors
```
Comment (howardjohn) [inline security/v1/jwt.proto:1]: We should be adding the `istio.io/sync-from` indicators like we have for networking protos so we don't need to manually keep them in sync.

**Review by howardjohn — COMMENTED** [review]
Can we split out new docs from the rest of the changes? They are ~unrelated and make it harder to review

File: security/v1/jwt.proto (hunk 1 above)
Comment (aryan16) [reply security/v1/jwt.proto]: done, thanks

**Review by howardjohn — APPROVED** [review]

**Review by linsun — CHANGES_REQUESTED** [review]
Only concern I have is the JWT.proto - some of the changes are too recent, for example ClaimToHeaders were changed few days ago and this feature is still relatively new IMHO, I don't feel comfortable promoting them to stable.  Others are fine.

Thoughts?

**Comment by aryan16** [comment]
cc : @liminw

**Review by linsun — APPROVED** [review]
Thanks @aryan16 for clarifying the claimtoheader. Given we don't have a better way to track each field other than the comment, LGTM

**Pull request merged**